rfcs/text/0014-extend-file-pe.md

```diff
@@ -77,17 +78,217 @@ Stage 1: Provide a high-level description of example sources of data. This does
 -->
-
+```
+"file": {
+  "pe": {
+    "machine_type": 332,
+    "flattened": {
+      "debug": [
+        {
+          "codeview": {
+            "signature": "RSDS",
+            "name": "G:\\Work\\new\\2345Pinyin\\bin\\Win32\\Release\\pdb\\2345PinyinSymbol.pdb",
+            "guid": "c27b9b78-caab-4629-9bf7-6acb4b63ac18",
+            "age": 1
+          },
+          "offset": 350324,
+          "size": 90,
+          "type_str": "IMAGE_DEBUG_TYPE_CODEVIEW",
+          "type": 2,
+          "timestamp": "Tue Jan 26 09:13:07 2021"
+        }
+      ],
+      "resources": [
+        {
+          "chi2": 257629.96875,
+          "filetype": "Data",
+          "entropy": 3.32720947265625,
+          "sha256": "49ba440033219598d7a9e5e27f5185063634c379dfaf4b188e49f0f43572beef",
+          "type": "RT_ICON",
+          "lang": "CHINESE SIMPLIFIED"
+        }
+      ]
+    },
+    "file_version": "7.4.0.8215",
+    "product": "2345王牌输入法",
+    "imports": [
+      {
+        "name": "GetStdHandle",
+        "library_name": "KERNEL32.dll",
+        "type": "function"
+      }
+    ],
+    "comments": "pinyin.2345.cc",
+    "overlay": {
+      "chi2": 26219.931640625,
+      "filetype": "Data",
+      "entropy": 7.256883144378662,
+      "offset": 480768,
+      "size": 14512,
+      "md5": "9a620f3583cc366f44e66df2a0f949a3"
+    },
+    "description": "2345王牌输入法-特殊符号输入器",
+    "creation_date": "2021-01-26T09:13:07.000Z",
+    "compilers": [
+      "id: 241, version: 40116 count=14",
+      "id: 243, version: 40116 count=153",
+      "id: 242, version: 40116 count=30",
+      "id: 259, version: 26706 count=25",
+      "id: 261, version: 26706 count=74",
+      "id: 260, version: 26706 count=35",
+      "id: 261, version: 27032 count=2",
+      "[ C ] VS2008 SP1 build 30729 count=5",
+      "[---] Unmarked objects count=344",
+      "[IMP] VS2008 SP1 build 30729 count=21",
+      "[---] Unmarked objects (old) count=1",
+      "id: 260, version: 27026 count=1",
+      "id: 265, version: 27026 count=131",
+      "id: 255, version: 27026 count=1",
+      "id: 151, version: 0 count=1",
+      "id: 258, version: 27026 count=1"
+    ],
+    "sections": [
+      {
+        "chi2": 1829004.63,
+        "virtual_address": 4096,
+        "entropy": 6.49,
+        "flags": "rx",
+        "name": ".text",
+        "raw_size": 260096,
+        "virtual_size": 259926,
+        "md5": "c001ed496e2cad5bb280d0d0bd97d4b4"
+      }
+    ],
+    "compile_timestamp": "2021-01-26T09:13:07.000Z",
+    "imphash": "12b9b9807742673c75c934de534f2014",
+    "original_file_name": "2345PinyinSymbol.exe",
+    "company": "版权所有(c) 2020, 2345移动科技",
+    "main_icon": {
+      "hash": {
+        "dhash": "e0c8f0b0b0acc071",
+        "md5": "8f852053f4dc483d5e25b2a663a4b0b4"
+      }
+    },
+    "entry_point": 165321,
+    "hash": {
+      "authentihash": "f489ddcf530ae777496441f1409989015ab9e32e57f5f5d7e51e6908ee1dc503"
```

Comment:
`authentihash` right now is defined at `pe.authentihash` and not `pe.hash.authentihash`.

Reply:
Questions caused by stale data examples. Refactoring.